README.md

```diff
@@ -140,6 +140,8 @@ If you want finer grained control about which polyfills you use, you'll have to
 
 ### Transpilation and native custom element gotchas
 
+*If you’re using Babel or some other tool to transpile your ES2015 code to ES5, you don’t need to worry about this if including `skatejs-web-components`.*
+
```

Comment:
One of the biggest questions I had was whether you *should* include `skatejs-web-components` (my impression now is that the answer to that is "yes, almost definitely"). Maybe reword to something like:

> If you’re using Babel or some other tool to transpile your ES2015 code to ES5, simply install `skatejs-web-components` and ignore the following.